qemu/tests/fio_perf.py

```diff
@@ -136,7 +114,22 @@ def fio_thread():
         """
         run fio command in guest
         """
-        session.cmd_status(run_cmd, cmd_timeout)
+        fio.run(run_fio_options)
+
+    def clean_tmp_files(session, os_type, guest_result_file, timeout):
+        """
+        del temporary files in guest
```

Comment:
Suggest `Clean temporary test result file, fio tool packages and fio processes after testing inside guest`